config/initializers/stripe.rb

```diff
@@ -0,0 +1,9 @@
+# Rails.configuration.stripe = {
+#   :publishable_key => "pk_test_cFzXGfqQNVCvN0FSkYTQm9zc",
+#   :secret_key      => "sk_test_kvwNXLfFJYQhr0zbZgb45vez"
+# }
+
+# Stripe.api_key = Rails.configuration.stripe[:secret_key]
+
+Stripe.api_key = "sk_test_kvwNXLfFJYQhr0zbZgb45vez"
```

Comment:
I think this is okay, but in the future, definitely know how to use `secrets.yml` or environment variables.